code/include/env/mstatush_32.h

```diff
@@ -0,0 +1,11 @@
+#ifndef RISCV_MSTATUSH_H
```

Comment:
As i said before, only use register names, no_32 anywhere, also remove any registers with _32 only. so the file name should be mstatush.h